backend/src/plugins/ReactionRoles/util/addMemberPendingRoleChange.ts

```diff
@@ -33,9 +33,7 @@ export async function addMemberPendingRoleChange(
           try {
             await member.roles.set(Array.from(newRoleIds.values()), "Reaction roles");
           } catch (e) {
-            logger.warn(
-              `Failed to apply role changes to ${renderUserUsername(member.user)} (${member.id}): ${e.message}`,
-            );
+            logger.warn(`Failed to apply role changes to ${renderUsername(member.user)} (${member.id}): ${e.message}`);
```

Comment:
Pretty sure there is a "renderUserInfo()" function somewhere that renders "${tag} ($i{d})"
```suggestion
            logger.warn(`Failed to apply role changes to ${renderUsername(member)} (${member.id}): ${e.message}`);
```